src/Makefile.am

```diff
@@ -99,7 +99,11 @@ BITCOIN_CORE_H = \
   compat/endian.h \
   compat/sanity.h \
   compressor.h \
+  consensus/consensus.h \
   consensus/merkle.h \
+  consensus/tx_verify.h \
```

Comment:
Need to also add `consensus/params.h` to this list.

For reference: TravisCI/Gitian builds first create a source tarball by running `make dist`, then continue to compile from that tarball (rather than from the `git clone` source tree). Since the aforementioned file isn't explicitly included here, it never ends up in the dist tarball, and the CI build fails with a missing file error.

Reply:
Ah cheers; i assumed the headers would get dragged in, will correct now.

Reply:
seems this got missed in your latest push

Reply:
Lets put this one down to operator error.. 😊 resolved.

Reply:
it happens, no worries :)